brave/brave-core
# Panel, tipping, and favicon should now be working correctly for twitch publishers

Fixes brave/brave-browser#1814
Fixes brave/brave-browser#2089
Fixes brave/brave-browser#1521

Rebased off of https://github.com/brave-intl/bat-native-ledger/pull/177

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Start Brave and navigate to twitch.tv
2. navigate to a publisher's live stream
3. Click BAT panel and ensure that tipping functions correctly
4. navigate to a publisher's live stream.
5. Select Videos from the channel headings and select a video-on-demand
6. Click BAT panel and ensure that tipping functions correctly
7. navigate to a publisher's live stream.
8. Select Videos from the channel headings and select a clip to play
9. Click BAT panel and ensure that tipping functions correctly
10. Repeat steps 1-9 for verified publishers and ensure that correct favicon displays

EDIT: Favicon should only show for verified publishers

## Reviewer Checklist:
